prod/westeurope/eucovidcert/functions_eucovidcert/subnet/terragrunt.hcl

```
@@ -0,0 +1,34 @@
+# Common
+dependency "virtual_network" {
+  config_path = "../../../common/virtual_network"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_subnet?ref=v3.0.3"
+}
+
+inputs = {
+  name = "fn3eucovidcert"
+
+  resource_group_name  = dependency.virtual_network.outputs.resource_group_name
+  virtual_network_name = dependency.virtual_network.outputs.resource_name
+  address_prefix       = "10.0.124.0/24"
```

```suggestion
  address_prefix       = "10.0.126.0/24"
```
126 is already in use